test/house_test.rb

```diff
@@ -0,0 +1,105 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+
+  def test_it_exists_and_has_attrs
```

Comment:
I would recommend splitting this into two tests. One for exists and one for attrs.